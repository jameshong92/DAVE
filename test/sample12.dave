// test for loop
int main() {
	int c = 0;
	for (c = 0; c < 4; c++) {
	    print("Hello World!");
	    int d;
	    for (d = 0; d < 2; d++) {
	        print("Hello Edwards!");
	        int e = 3;
	        for (e=3; e > 0; e--) {
                print("Hello Stephen!");
	        }
	    }
	}
}