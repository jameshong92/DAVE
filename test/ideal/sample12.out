Hello World!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
Hello World!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
Hello World!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
Hello World!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
Hello Edwards!
Hello Stephen!
Hello Stephen!
Hello Stephen!
